Makefile

```diff
@@ -1,17 +1,17 @@
-versionDatos = 1b981ee4293a
+run: build
+	docker-compose up
 
 # Archivo con los datos de la cantidad de nidos en las islas del Pacífico
-archivoDatos = \
-	data/base_datos_parejas_aves_marinas_islas.xlsx
+archivoDatos = api-datos/data/base_datos_parejas_aves_marinas_islas.xlsx
+versionDatos = 1b981ee4293a
+
+$(archivoDatos):
+	mkdir --parents $(@D)
+	curl --output $@ --user $${BITBUCKET_USERNAME}:$${BITBUCKET_PASSWORD} https://bitbucket.org/IslasGECI/datos-binarios/raw/$(versionDatos)/$(@F)
```

Comment:
Entre paréntesis es porque es una variable del Makefile, aquí toma la parte del directorio que viene en la variable. Si mi memoria no me falla cuando usas llaves te refieres a una variable de entorno 🤔.

Reply:
Entre paréntesis es porque es una variable del Makefile, aquí toma la parte del directorio que viene en la variable. Si mi memoria no me falla cuando usas llaves te refieres a una variable de entorno 🤔.

Reply:
> To substitute a variable’s value, write a dollar sign followed by the name of the variable in parentheses or braces: either ‘$(foo)’ or ‘${foo}’ is a valid reference to the variable foo. This special significance of ‘$’ is why you must write ‘$$’ to have the effect of a single dollar sign in a file name or recipe.

- [Basics of Variable References](https://www.gnu.org/software/make/manual/html_node/Reference.html)

Reply:
Checa @nepito lo que dice este tal [Etan Reisner](https://stackoverflow.com/questions/25185607/whats-the-difference-between-parenthesis-and-curly-bracket-syntax-in-ma#comment39217556_25185742):

> I use the $() in make to avoid causing myself confusion (more than already exists) between make and shell variables.